pootle/apps/pootle_store/models.py

```diff
@@ -1720,6 +1745,8 @@ def update(self, overwrite=False, store=None, only_newer=False):
             the current DB store's FS store will be used as a source.
         :param only_newer: Whether to update only the files that changed on
             disk after the last sync.
+        :param user: User to attribute updates to.
+        :param submission_type: Submission type of saved updates.
```

Comment:
It is easy but takes some time. Just do an interactive rebase for the commit where the changes should go, add the changes to the commit and finish interactive rebase. Then force push the PR branch.